pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -20,12 +20,12 @@ callPackage ./generic.nix args {
   # IMPORTANT: Always use a tagged release candidate or commits from the
   # zfs-<version>-staging branch, because this is tested by the OpenZFS
   # maintainers.
-  version = "2.2.6";
-  # rev = "";
+  version = "2.3.0-rc2";
+  rev = "zfs-2.3.0-rc2";
```

Comment:
On line 17, `kernelCompatible = kernel: kernel.kernelOlder "6.11";` should be anything before 6.12 as 2.3.0-rc2 supports 6.11